sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/TableAttributeBindingProvider.cs

```diff
@@ -39,6 +39,12 @@ private IBinding TryCreate(BindingProviderContext context)
                 return null;
             }
 
+            // ParameterBindingData and ParameterBindingData[] are bound by separate binding methods using BindToInput<> specific to those respective types
```

Comment:
I think you can just merge these checks into the JArray case above, since all 3 of these types are handled by rule based binders (BindToInput). So a single if block with a general comment like "These types are handled by rule-based binders".